src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figures;

```

Comment:
```suggestion
package core.basesyntax.figure;
```
Use singular for package names. Also, you can name this package "model"